RELEASING.md

```diff
@@ -0,0 +1,15 @@
+# Releasing
+
+- Use `bump2version`, available in the installed project dependencies, to update the version number. For example, to bump from v1.1.1 to the next patch version:
+
+```shell
+> bump2version patch                  # 1.1.1 -> 1.1.2-dev0
+> bump2version --allow-dirty release  # 1.1.2-dev0 -> 1.1.2
```

Comment:
I think this is better. Combined with the change to `.bumpversion.cfg` that turns off auto-commit, these steps produce changes in your working directory ready for your review and choice to add to a commit.